lib/components/Masthead/Masthead.tsx

```diff
@@ -117,6 +147,7 @@ export class Masthead extends React.PureComponent<MastheadProperties> {
                         </InlinePopup.Panel>
                     </InlinePopup.Container>
                 }
+                {logo && <Attr.div key={'masthead-logo'} className={cx('masthead-logo', { 'force-hide-search': expanded })}>{logo}</Attr.div>}
                 <Attr.span key={'masthead-branding'} className={cx('masthead-branding', 'inline-text-overflow', { 'force-hide-search': expanded })}>{this.props.branding}</Attr.span>

```

Comment:
change `this.props.branding` to `branding` since you are destructuring it in line 122

Reply:
Good catch